src/test/java/org/example/P19_rectanglePerimeterTest.java

```diff
@@ -0,0 +1,17 @@
+package org.example;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class P19_rectanglePerimeterTest {
+    @Test
+    public void perimeter() {
+
+        int width = 5;
+        int height = 6;
+        int sum = P19_rectanglePerimeter.sum(width, height);
```

Comment:
this variable should be called `perimeter` not `sum`

Reply:
done

Reply:
modified